src/main/java/io/confluent/connect/hdfs/tools/SchemaSourceTask.java

```diff
@@ -41,6 +47,9 @@
   private static final Logger log = LoggerFactory.getLogger(SchemaSourceTask.class);
   private static final String ID_FIELD = "id";
   private static final String SEQNO_FIELD = "seqno";
+  private static final String SCHEMA_TYPE = "schema.type";
+  private static final String CONNECT_AVRO_DECIMAL_PRECISION_PROP = "connect.decimal.precision";
```

Comment:
Where is this config originally declared? Can we import the member variable directly here instead of just the string value?